pkg/test/framework/components/istio/operator.go

```diff
@@ -92,7 +92,6 @@ var leaderElectionConfigMaps = []string{
 	leaderelection.IngressController,
 	leaderelection.NamespaceController,
 	leaderelection.ValidationController,
-	leaderelection.StatusController,
```

Comment:
This change doesn't seem related?

Reply:
Good catch.. yes its not I got a bit ambitious on this and fixed another issue (https://github.com/istio/istio/issues/24524) but should probably split it out